src/explore-education-statistics-frontend/test/setupTests.js

```diff
@@ -13,3 +13,6 @@ if (typeof window !== 'undefined') {
   // fetch polyfill for making API calls.
   require('cross-fetch');
 }
+
+global.Request = jest.requireActual('node-fetch').Request;
+global.Response = jest.requireActual('node-fetch').Response;
```

Comment:
Not sure we need this as Fetch API landed in Node in v18 (which we're currently on).

Reply:
Without it the tests fail to run because of `ReferenceError: Request is not defined`. I've checked that my local node version is 18.

Reply:
Hmm weird, the API documentation seems to suggest it should be there! Guess this is fine anyway